distribution/docs/src/main/resources/content/_managing/attributes-added-by-input-transformers.adoc

```diff
@@ -0,0 +1,28 @@
+:title: Attributes Added by Input Transformers
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How input tranformers add attributes to metacards.
+:order: 01
+
+=== {title}
+
+<<_available_input_transformers,Input Transformers>> create and populate metacards from a resource.
+
+{branding} chooses which input transformer to use by:
+
+. Resolving the mimetype for the resource. This is done by trying the following steps in order until a proper mimetype is found:
+  .. Extracting an optionally given mimetype with the resource.
+  .. Passing the resource's file extension to a <<_mime_type_mapper,Mimetype Mapper>>.
+  .. Using https://tika.apache.org/[Apache Tika].
+  .. Manually parsing the resource's file contents for the first character (for example: if the file starts with `<`, it is assumed to be XML).
+. Gathering all of the input transformers associated with the resolved mimetype.
+. Iterating through the transformers until a successful transformation is performed.
+
+The first transformer that can successfully create a metacard from the ingested resource is chosen. If no transformer can successfully transform the resource, the ingestion process fails.
+
+There are a couple of exceptions to this strategy, dependent on the method of ingest:
+
+* Single resource ingest through <<_using_intrigue,Intrigue>> only performs the first step of resolving the mimetype.
+* The in-place <<_configuring_the_content_directory_monitor,Content Directory Monitor>> can be passed an optional `transformerId` to use. If not, it only performs the first step of resolving the mimetype, and defaults to `application/octet-stream` if no mimetype could be found.
```

Comment:
✏️ I am not sure if this is worth mentioning. This is not something that is configurable by an Admin. Instead the in place CDM itself would rely on file extension and/or file itself to determine the MIME type.